test/syntax.js

```diff
@@ -57,63 +54,6 @@ describe('Azure swagger schema validation:', function () {
     });
   }).value();
 
-  describe('Azure composite swagger schema validation:', function () {
```

Comment:
Ahha good point. To run the semantic validation on config file, we need to use autorest plugin instead of calling into oav directly. Let me look for that and see how that works out.